vsts/node-nightly-df.yaml

```diff
@@ -80,7 +84,9 @@ phases:
     env:
       IOTHUB_CONNECTION_STRING: $(WINDOWS-IOTHUB-CONNECTION-STRING)
 
-  - script: 'lerna run e2e'
+  - script: |
+       call lerna run e2e
+       call dir e2etests
```

Comment:
>call dir e2etests [](start = 7, length = 17)

please don't check this in

Reply:
Certainly won't.  Was just trying to expirement to find out why the report code was failing.  returned to original.

---
In reply to: [221354232](https://github.com/Azure/azure-iot-sdk-node/pull/378#discussion_r221354232) [](ancestors = 221354232)